lib/pure/math.nim

```diff
@@ -62,6 +62,9 @@ when defined(c) or defined(cpp):
   proc c_isnan(x: float): bool {.importc: "isnan", header: "<math.h>".}
     # a generic like `x: SomeFloat` might work too if this is implemented via a C macro.
 
+  proc c_copysign(x, y: float32): float32 {.importc: "copysignf", header: "math.h".}
```

Comment:
* float32 is incorrect, should be cfloat
* float64 is incorrect, should be cdouble
(I made same mistake with c_isnan; we should check all math procs at least)

note that those aren't necessarily same types